tests/manual-test-cases/Group5-Functional-Tests/5-18-Datastore-Cluster-SDRS.robot

```diff
@@ -0,0 +1,62 @@
+*** Settings ***
+Documentation  Test 5-18 - Datastore Cluster SDRS
+Resource  ../../resources/Util.robot
+Test Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+
+*** Test Cases ***
+SDRS Datastore
+    Pass Execution  VIC does not support SDRS yet, see issue #2729
+    ${out}=  Deploy Nimbus Testbed  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}  --customizeTestbed '/esx desiredPassword=e2eFunctionalTest' --noSupportBundles --vcvaBuild ${VC_VERSION} --esxBuild ${ESX_VERSION} --testbedName vcqa-sdrs-fc-fullInstall-vcva --runName vic-fc
+    Set Global Variable  @{list}  %{NIMBUS_USER}-vic-fc.vcva-${VC_VERSION}  %{NIMBUS_USER}-vic-fc.esx.0  %{NIMBUS_USER}-vic-fc.esx.1  %{NIMBUS_USER}-vic-fc.fc.0
+    Should Contain  ${out}  "deployment_result"=>"PASS"
+
+    ${out}=  Execute Command  nimbus-ctl ip %{NIMBUS_USER}-vic-fc.vcva-${VC_VERSION} | grep %{NIMBUS_USER}-vic-fc.vcva-${VC_VERSION}
+    ${vc-ip}=  Fetch From Right  ${out}  ${SPACE}
+    
+    ${out}=  Execute Command  nimbus-ctl ip %{NIMBUS_USER}-vic-fc.esx.0 | grep %{NIMBUS_USER}-vic-fc.esx.0
+    ${esx0-ip}=  Fetch From Right  ${out}  ${SPACE}
+    
+    ${out}=  Execute Command  nimbus-ctl ip %{NIMBUS_USER}-vic-fc.esx.1 | grep %{NIMBUS_USER}-vic-fc.esx.1
+    ${esx1-ip}=  Fetch From Right  ${out}  ${SPACE}
+
+    Set Environment Variable  GOVC_URL  ${esx0-ip}
+    Set Environment Variable  GOVC_USERNAME  root
+    Set Environment Variable  GOVC_PASSWORD  e2eFunctionalTest
+    Run  govc host.esxcli network firewall set -e false
+    Set Environment Variable  GOVC_URL  ${esx1-ip}
+    Run  govc host.esxcli network firewall set -e false
+
+    Log To Console  Set environment variables up for GOVC
+    Set Environment Variable  GOVC_URL  ${vc-ip}
+    Set Environment Variable  GOVC_USERNAME  Administrator@vsphere.local
+    Set Environment Variable  GOVC_PASSWORD  Admin\!23
+
+    Create A Distributed Switch  vcqaDC
+
+    Create Three Distributed Port Groups  vcqaDC
+
+    Add Host To Distributed Switch  /vcqaDC/host/cls
+
+    ${out}=  Run  govc folder.create -pod=true /vcqaDC/datastore/sdrs
+    ${out}=  Run  govc object.mv /vcqaDC/datastore/sharedVmfs-0 /vcqaDC/datastore/sdrs
```

Comment:
Note that you can collapse these 4 to a single call like so: `govc object.mv /vcqaDC/datastore/sharedVmfs-* /vcqaDC/datastore/sdrs`